fairlearn/datasets/_fetch_adult.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import os
+import pathlib
+
+from sklearn.datasets import fetch_openml
+
+
+def fetch_adult(*, cache=True, data_home=None,
+                as_frame=False, return_X_y=False):
+    """Load the UCI Adult dataset (binary classification).
+
+    Download it if necessary.
+
+    ==============   ==============
+    Samples total             48842
+    Dimensionality               14
+    Features                   real
+    Classes                       2
+    ==============   ==============
+
+    Source: https://archive.ics.uci.edu/ml/datasets/Adult
+    Ronny Kohavi and Barry Becker, "Scaling Up the Accuracy of Naive-Bayes
+    Classifiers: a Decision-Tree Hybrid", Proceedings of the Second
+    International Conference on Knowledge Discovery and Data Mining, 1996
+
+    Prediction task is to determine whether a person makes over $50,000 a
+    year.
+
+    Parameters
+    ----------
+    cache : boolean, default=True
+        Whether to cache downloaded datasets using joblib.
+
+    data_home : optional, default: None
+        Specify another download and cache folder for the datasets. By default
+        all scikit-learn data is stored in '~/fairlearn_data' subfolders.
+
+    as_frame : boolean, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+        The Bunch will contain a ``frame`` attribute with the target and the
+        data. If ``return_X_y`` is True, then ``(data, target)`` will be pandas
+        DataFrames or Series as describe above.
+
+    return_X_y : boolean, default=False.
```

Comment:
Is this a common name for this argument? I don't think it makes much sense.

Reply:
I copied the structure from fetch_boston. It does make sense to me what it means and does, although I would not really use it since it’s a subset of what you get otherwise.

Reply:
... and yes, this is the arg name in openml.